WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -137,4 +137,33 @@ class LoginTests: XCTestCase {
             .tryProceed(password: "invalidPswd")
             .verifyLoginError()
     }
+
+    //Self-Hosted after WordPress.com login
+    //Login to a WordPress.com account, then add a self-hosted site.
+    func testAddSelfHostedSiteAfterWPcomLogin() {
+        _ = PrologueScreen().selectContinue()
```

Comment:
I have a warning here:
```
Using '_' to ignore the result of a Void-returning function is redundant
```

Usually `_ =` used to silence another warning, when function returns something and it's ignored. Try removing it in `testUnsuccessfulLogin()` above, for example. Here we don't need it here because last function in the chain (`removeSelfHostedSite()`) doesn't return anything (= returns Void).

Reply:
Ah, I see! When I remove this from `testUnsuccessfulLogin()`, I get a warning that `Result of call to 'verifyLoginError()' is unused`. Because `verifyLoginError()` does return something (the LoginPasswordScreen), we expect to use the return.
And `_ =` indicates we're not going to use the return, so there's no need for the warning.